HTAN.model.csv

```diff
@@ -122,7 +122,7 @@ GeoMx DSP Workflow Parameter Description,Parameters used to run the GeoMx DSP wo
 GeoMx DSP Workflow Link,Link to workflow or command. DockStore.org recommended. URL,,,,FALSE,Spatial Transcriptomics,,,
 NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,GeoMx ROI and Segment Metadata Attributes. The assayed biospecimen should be reported one per row with the associated ROI coordinates. ,,"HTAN Parent Biospecimen ID, Scan name, ROI name, Segment name, ROI X Coordinate,ROI Y Coordinate, Tags, QC status, Scan Height, Scan Width, Scan Offset X, Scan Offset Y, Binding Density, Positive norm factor, Surface area, Nuclei count, Tissue Stain",,FALSE,Assay,,,
 Scan name,GeoMx Scan name (as appears in Segment Summary),,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
-ROI name,"ROI name (application generated). For Xenium this is referred to as the “region name”",,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
+ROI name,ROI name (application generated). For Xenium this is referred to as the “region name”,,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,

```

Comment:
Here and below these are all OK as they are just dropping superfluous quotes per the CSV linter